website/docs/components/time/partials/code/how-to-use.md

```diff
@@ -0,0 +1,43 @@
+!!! Info
+
+The Time component exists only in code. The Figma equivalent would be a manually input text object with a formatted string.
```

Comment:
I'm not sure if this is the place to do it, but it might be worth cross linking to the soon to come content section.

```suggestion
The Time component exists only in code. The Figma equivalent would be a manually input text object with a formatted string. For writing guidelines related to this component, read our content [Date and Time](content/writing-guidelines#date-and-time) section.
```